readme.md

```diff
@@ -55,6 +76,7 @@ console.log(results);
 Your Answer:
 ```text
```

Comment:
Let me know if you have any questions about #4. Hint: it has to do with asynchronicity.